tests/functional/builtins/codegen/test_abi_decode.py

```diff
@@ -486,10 +486,10 @@ def f(x: Bytes[32 * 3]):
     assert decoded_y1 != decoded_y2
     """
     c = get_contract(code)
-    data = (0xd45754f8).to_bytes(4, "big")
+    data = (0xD45754F8).to_bytes(4, "big")
     data += (0x20).to_bytes(32, "big")
     data += (0x60).to_bytes(32, "big")
-    data += (0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffa0).to_bytes(32, "big")
+    data += (0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFA0).to_bytes(32, "big")
```

Comment:
is there something else we can do about this constant? maybe `(2**256 - 0x60).to_bytes(...)`